metron-analytics/metron-profiler/src/main/java/org/apache/metron/profiler/bolt/ProfileSplitterBolt.java

```
@@ -0,0 +1,158 @@
+import backtype.storm.task.OutputCollector;
+import backtype.storm.task.TopologyContext;
+import backtype.storm.topology.OutputFieldsDeclarer;
+import backtype.storm.tuple.Fields;
+import backtype.storm.tuple.Tuple;
+import backtype.storm.tuple.Values;
+import org.apache.metron.common.bolt.ConfiguredProfilerBolt;
+import org.apache.metron.common.configuration.profiler.ProfileConfig;
+import org.apache.metron.common.configuration.profiler.ProfilerConfig;
+import org.apache.metron.profiler.stellar.StellarExecutor;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.UnsupportedEncodingException;
+import java.util.Map;
+
+/**
+ * The bolt responsible for filtering incoming messages and directing
+ * each to the one or more bolts responsible for building a Profile.  Each
+ * message may be needed by 0, 1 or even many Profiles.
+ */
+public class ProfileSplitterBolt extends ConfiguredProfilerBolt {
+
+  protected static final Logger LOG = LoggerFactory.getLogger(ProfileSplitterBolt.class);
+
+  private OutputCollector collector;
+
+  /**
+   * JSON parser.
+   */
+  private transient JSONParser parser;
+
+  /**
+   * Executes Stellar code.
+   */
+  private StellarExecutor executor;
+
+  /**
+   * @param zookeeperUrl The Zookeeper URL that contains the configuration for this bolt.
+   */
+  public ProfileSplitterBolt(String zookeeperUrl) {
+    super(zookeeperUrl);
+  }
+
+  @Override
+  public void prepare(Map stormConf, TopologyContext context, OutputCollector collector) {
+    super.prepare(stormConf, context, collector);
+    this.collector = collector;
+    this.parser = new JSONParser();
+  }
+
+  @Override
+  public void execute(Tuple input) {
+    try {
+      doExecute(input);
+
+    } catch (IllegalArgumentException | ParseException | UnsupportedEncodingException e) {
+      LOG.error("Unexpected exception", e);
```

Can you bubble the exception message up like `"Unexpected exception: e.getMessage()`?